.github/workflows/ci.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
               run: yarn lint
             - name: Check types compile
               run: yarn type-check
-            - name: Check tests pass
-              run: yarn test
+            # - name: Check tests pass
+            #   run: yarn test
```

Comment:
I think we commented this out because at some point we didn't have any tests and this was failing because for some reason it fails when there are no tests at all.